### ISSUE TITLE
pan up/down does the opposite: slightly annoying

### ISSUE THREAD

**Issue by hugoledoux** [fhb1990/CityJSON-viewer#13]
is that what you had in mind?

**Comment by Athelena** [comment]
Yeah it would be nice to scroll left, right, up and down and to chose which part of the model to zoom into. Unless this is already possible then it would be nice to get some instructions on how to do it.

**Comment by hugoledoux** [comment]
hold cmd and mouse

**Comment by Athelena** [comment]
Fair enough, but only works left to right, not up and down
